### ISSUE TITLE
Geocalc.DMS.t() not compiling on Geocalc 0.8.0, Elixir 1.10.0

### ISSUE THREAD

**Issue by mjquinlan2000** [yltsrc/geocalc#55]
I'm getting a compilation error for the type Geocalc.DMS.t() on Elixir 1.10:

```
== Compilation error in file lib/geocalc/dms.ex ==
** (CompileError) lib/geocalc/dms.ex:18: type t/0 undefined (no such type in Geocalc.DMS)
    (elixir 1.10.0) lib/kernel/typespec.ex:898: Kernel.Typespec.compile_error/2
    (stdlib 3.11.2) lists.erl:1354: :lists.mapfoldl/3
    (elixir 1.10.0) lib/kernel/typespec.ex:950: Kernel.Typespec.fn_args/5
    (elixir 1.10.0) lib/kernel/typespec.ex:936: Kernel.Typespec.fn_args/6
    (elixir 1.10.0) lib/kernel/typespec.ex:377: Kernel.Typespec.translate_spec/8
    (stdlib 3.11.2) lists.erl:1354: :lists.mapfoldl/3
    (elixir 1.10.0) lib/kernel/typespec.ex:229: Kernel.Typespec.translate_typespecs_for_module/2
```

I see it's already been updated on the master branch. Can we get this into the hex packages? v0.7.1 is the most recent version that will compile for me.

Thanks

**Comment by nahiyan** [comment]
Same here, 0.7.2 is the most recent version which works for me. Getting the exact same errors on the later versions.

**Comment by astutecat** [comment]
I am also seeing this issue, keen to see it released out on hex.

**Comment by rockneurotiko** [comment]
@yltsrc  Could you do a release so we have the type fix in hex? 🙏

**Comment by yltsrc** [comment]
no issue on travis with elixir 1.10.3 and otp 21.3, 22.3 https://travis-ci.org/github/yltsrc/geocalc/jobs/693753313
can somebody confirm that the issue is still present or fixed?
if it is not fixed yet, then I need help with reproducing the issue, because I have the same result as on travis

**Comment by rockneurotiko** [comment]
@yltsrc it was fixed on this MR https://github.com/yltsrc/geocalc/pull/54 but there are no release on hex with the fix

**Comment by astutecat** [comment]
Just to be clear, there's been changes since `0.8.0` was published, which resolves this issue - You need to bump the version in minx.exs and then publish to hex.

**Comment by yltsrc** [comment]
fixed with #56 
thanks for your help
